packages/core/src/awsService/appBuilder/serverlessLand.ts

```diff
@@ -0,0 +1,131 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+import * as path from 'path'
+// import * as vscode from 'vscode'
+import { getTelemetryResult, RegionProvider } from '../../shared'
+import { getLogger } from '../../shared/logger'
+import globals from '../../shared/extensionGlobals'
+import { checklogs } from '../../shared/localizedText'
+// import { fileExists } from '../../shared/filesystemUtilities'
+// import { CreateServerlessLandWizardForm } from '../appBuilder/wizards/serverlessLandWizard'
+import { Result, telemetry } from '../../shared/telemetry/telemetry'
+import { CreateServerlessLandWizard } from '../appBuilder/wizards/serverlessLandWizard'
+import { ExtContext } from '../../shared/extensions'
+import { addFolderToWorkspace } from '../../shared/utilities/workspaceUtils'
+
+export const readmeFile: string = 'README.md'
+
+// export async function getProjectUri(
+//     config: Pick<CreateServerlessLandWizardForm, 'location' | 'name'>,
+//     files: string
+// ): Promise<vscode.Uri | undefined> {
+//     if (files.length === 0) {
+//         throw Error('expected "files" parameter to have at least one item')
+//     }
+//     let file: string
+//     let cfnTemplatePath: string
+//     for (const f of files) {
+//         file = f
+//         cfnTemplatePath = path.resolve(config.location.fsPath, config.name, file)
+//         if (await fileExists(cfnTemplatePath)) {
+//             return vscode.Uri.file(cfnTemplatePath)
+//         }
+//     }
+//     void vscode.window.showWarningMessage(
+//         localize(
+//             'AWS.samcli.initWizard.source.error.notFound',
+//             'Project created successfully, but {0} file not found: {1}',
+//             file!,
+//             cfnTemplatePath!
+//         )
+//     )
+// }
```

Comment:
Any reason to add this code, but commented out? I feel like if it's not ready, then you can leave it out completely, otherwise I'm not sure if I'm expected to review it now or not. (same with the similar commented code below)

Reply:
It was not ready so I commented out. I have removed it from here and will open a new PR for it.